config/boot/entities.rb

```diff
@@ -2,6 +2,8 @@
   init do
     require_relative(File.join(Inferno::Application.root, 'lib', 'inferno', 'exceptions'))
 
+    require_relative(File.join(Inferno::Application.root, 'lib', 'inferno', 'entities', 'attributes.rb'))
+    require_relative(File.join(Inferno::Application.root, 'lib', 'inferno', 'entities', 'entity.rb'))
```

Comment:
I might just be blind, but it looks like line 7 should be importing these files (and all other files in `lib/inferno/entities`. Do these need to be manually loaded up first?

Reply:
It seems like hte other files in `entities` inconsistently import `attributes`. e.g. both `request.rb` and `result.rb` use `attributes`, but only only `result` explicitly imports it.

For consistency I think it would make sense to only import here at `boot` or consistently make sure that any file/class/module that uses another requires it.

Reply:
`attributes` and `entities` are used by the other entities. They also just happen to be the first files alphabetically in their folder. I think mac and windows load files alphabetically, so they're loaded before anything that uses them is loaded, but that isn't the case in linux. I'll check to make sure I can get rid of the requires in the other entities.

Reply:
Yup, those other requires are no longer needed.